fix (ui5-time-picker): Can close the popup with keybord

a-z-ivanov opened the pull request:
**Thank you for your contribution!** 👏

To get it merged faster, kindly review the checklist below:

## Pull Request Checklist
- [ ] Reviewed the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents/blob/master/CONTRIBUTING.md)
    + Especially the [How to Contribute](https://github.com/SAP/ui5-webcomponents/blob/master/CONTRIBUTING.md#how-to-contribute) section 
- [ ] [Correct commit message style](https://github.com/SAP/ui5-webcomponents/blob/master/docs/Guidelines.md#commit-message-style)

ilhan007 reviewed (changes requested):
Alex could you give us more backgound, is this fix related to reported issue? If yes, please link it (via fixes: issue). Or if not, couple of words in the commit msg body would be nice.

Currently, the build fails due to a single test in the TimePicker:

TimePicker.spec.js
✖ tests change event
Change fired as expected: expected '2' to equal '1'

a-z-ivanov commented:
> Alex could you give us more backgound, is this fix related to reported issue? If yes, please link it (via fixes: issue). Or if not, couple of words in the commit msg body would be nice.
> 
> Currently, the build fails due to a single test in the TimePicker:
> 
> TimePicker.spec.js ✖ tests change event Change fired as expected: expected '2' to equal '1'

Fixed the test. Added a new one. Added one sentence in the commit message.

ilhan007 commented:
@tsanislavgatev could you perform a code review from @SAP/ui5-webcomponents-topic-b side
